app-emulation/fuse-utils/fuse-utils-1.4.3.ebuild

```diff
@@ -1,7 +1,9 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
This requires the ebuilds to be revbumped. Just add the patch files to existing ebuilds, EAPI-6 is still perfectly valid to be used.